app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Users::ConfirmationsController < Devise::ConfirmationsController
```

Comment:
Why copying the controller and commenting the methods? Shouldn't this be controlled in the user model? (:confirmable is not activated) https://github.com/singularities/circular-works/blob/master/app/models/user.rb#L3